app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,8 @@ def create
     if user&.authenticate params[:session][:password]
       if user.activated?
         log_in user
-        params[:session][:remember_me] == Settings.users.remember ? remember(user) : forget(user)
+        params[:session][:remember_me] == Settings.users.remember ?
+                                          remember(user) : forget(user)
```

Comment:
xuống dòng thì ko dùng toán tử 3 ngôi đâu e nha, mà phải dùng if .. else
tuy nhiên chỗ này so sánh giá trị "1" thì ko cần phải setting đâu, vì giá trị này sẽ không thay đổi

Reply:
chị Vân bảo đưa 1 vào setting thì theo ai đây c :smiley: